lib/menu.rb

```diff
@@ -0,0 +1,25 @@
+class Menu
+
+attr_reader :print_menu, :todays_menu
+
+    def initialize
+      @print_menu = []
+      @todays_menu = []
+    end
+
+    def todays_menu
```

Comment:
This overrides the `attr_reader :todays_menu`, which is therefore not required.  Note though that this method exposes the internal array to the public, which breaks encapsulation.